### PR TITLE
document steps for configuring okta provider

### PR DESCRIPTION
**Pull request by punkle**

### CODE REVIEW

File: content/docs/integrations/okta/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+humanName: Okta
+integrationType: OSS plugin
+---
+
+## Introduction
+
+In order to use the Okta organization entity plugin, Roadie needs an API token, and the organization url to communicate with Okta APIs.
+
+This page describes how to create and set up the API token.
```
Comment (jorgelainfiesta) [inline content/docs/integrations/okta/index.md:14]:
```suggestion
In order to use the Okta organization plugin, Roadie needs an Okta API token, and the organization URL to communicate with Okta APIs. On this page, you'll learn how to set up this API token in your Roadie Admin panel. 
```

File: content/docs/integrations/okta/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Prerequisites
+You will need to [configure an Okta API token](https://developer.okta.com/docs/guides/create-an-api-token/) for use by Backstage.
```
Comment (jorgelainfiesta) [inline content/docs/integrations/okta/index.md:17]:
```suggestion
Obtain an [Okta API token](https://developer.okta.com/docs/guides/create-an-api-token/) to use with Roadie Backstage. 
```

Also, I suggest you change this from "Prerequisites" to "Step 1"

File: content/docs/integrations/okta/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Steps
+
+### Step 1: Set the Okta API token
+
+Visit the secrets page in the Backstage UI and set the `OKTA_TOKEN` secret.
```
Comment (jorgelainfiesta) [inline content/docs/integrations/okta/index.md:23]:
```suggestion
Set the `OKTA_TOKEN` secret in the secrete manager of your Roadie Backstage Admin panel, located in the following path:
```

File: content/docs/integrations/okta/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+```text
+https://<tenant-name>.roadie.so/administration/settings/secrets
+```
+
+### Step 2: Set the Okta Organization URL
+
+Visit the Okta configuration page in Backstage and set the Organization URL.
```
Comment (jorgelainfiesta) [inline content/docs/integrations/okta/index.md:31]:
```suggestion
Set the Okta Organization URL in the Okta configuration page of your Roadie Backstage Admin panel, located in the following path:
```

**Review by jorgelainfiesta — COMMENTED** [review]
Thanks for documenting this, here are a few copy suggestions for your consideration.

File: content/docs/integrations/okta/index.md (hunk 2 above)
Comment (punkle) [reply content/docs/integrations/okta/index.md]: done. thanks

File: content/docs/integrations/okta/index.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+publishedDate: '2022-08-02T17:00:00.0Z'
+description: How to configure Roadie to collect users from Okta.
+
+humanName: Okta
+integrationType: OSS plugin
+---
+
+## Introduction
+
+In order to use the Okta organization plugin, Roadie needs an Okta API token, and the organization URL to communicate with Okta APIs. On this page, you'll learn how to set up this API token in your Roadie Admin panel.
```
Comment (jorgelainfiesta) [inline content/docs/integrations/okta/index.md:13]:
```suggestion
In order to use the Okta organization plugin, Roadie needs an Okta API token, and the organization URL to communicate with Okta's APIs. On this page, you'll learn how to set up these two fields in your Roadie Admin panel. 
```

**Review by jorgelainfiesta — APPROVED** [review]
Thanks, added a final edit that I missed

**Pull request merged**